src/LabLog.Tests/ReadTests/Steps/Steps.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Linq;
+using CheetahTesting;
+using LabLog.Domain.Entities;
+using LabLog.Domain.Events;
+using Xunit;
+
+namespace LabLog.ReadTests.Steps
+{
+    public static class Steps
+    {
+        public static void RoomCreatedEvent(this IGiven<ReadRoomContext> given)
+        {
+            RoomCreatedEvent roomCreatedEvent = new RoomCreatedEvent();
+            roomCreatedEvent.Name = "Test Room";
+            LabEvent labEvent = LabEvent.Create(new Guid("11111111-1111-1111-1111-11111111"), 1, roomCreatedEvent); 
+            given.Context.RetrievedEvents.Add(labEvent);
+        }
+
+        public static void CreateTheRoom(this IWhen<ReadRoomContext> when)
+        {
+            ILabEvent labEvent = when.Context.RetrievedEvents[0];
+            when.Context.Room = new RoomModel();
+            RoomCreatedEvent roomCreatedEvent= labEvent.GetEventBody<RoomCreatedEvent>();
+            when.Context.Room.Name = roomCreatedEvent.Name; //this won't work because of lack of event handler. Check room.cs line 38
```

Comment:
you're doing manual manipulation of the Room here in a test... this is the logic that you're supposed to be testing, so doing it in a test like this makes no sense.
This should be `when.Context.Room.ReplayEvent(labEvent)`

Reply:
Oh yeah, I knew I'd got lost somewhere with what I was doing :P I started with manually creating the event, and got carried away!